File: task-03/sub-task03/Javascript/pattern.js
function printPattern(rows) {
    for (let i = 0; i < rows; i++) {
      for (let j = 0; j < rows - i; j++) {
        process.stdout.write(" ");
      }
      for (let k = 0; k < 2 * i + 1; k++) {
        process.stdout.write("*");
      }
      console.log();
    }
  
    for (let i = rows - 2; i >= 0; i--) {
      for (let j = 0; j < rows - i; j++) {
        process.stdout.write(" ");
      }
      for (let k = 0; k < 2 * i + 1; k++) {
        process.stdout.write("*");
      }
      console.log();
    }
  }
  
  const rows = 3;
  printPattern(rows);

File: task-03/sub-task03/Rust/pattern.rs
use std::io::{stdin, stdout, Write};

fn print_pattern(rows: i32) {
  for i in 0..rows {
    for j in 0..rows - i {
      print!(" ");
    }
    for k in 0..2 * i + 1 {
      print!("*");
    }
    println!();
  }

  for i in (rows - 2)..=0 {
    for j in 0..rows - i {
      print!(" ");
    }
    for k in 0..2 * i + 1 {
      print!("*");
    }
    println!();
  }
}

fn main() {
  let mut rows = String::new();
  stdout().write_all(b"Enter the number of rows: ").unwrap();
  stdin().read_line(&mut rows).expect("Failed to read line");
  let rows: i32 = rows.trim().parse().expect("Invalid input");
  print_pattern(rows);
}